src/main/java/org/apache/zeppelin/interpreter/util/ProcessLauncher.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * The MIT License
```

Comment:
We actually cannot re-license it, even though the licenses are compatible.  I did it this way just because I didn't want to bother looking for the skip flag for RAT, nor test if AL header would work. :clock1: